internal/config/identity/ldap/config.go

```diff
@@ -459,7 +459,9 @@ func Enabled(kvs config.KVS) bool {
 
 // Lookup - initializes LDAP config, overrides config, if any ENV values are set.
 func Lookup(kvs config.KVS, rootCAs *x509.CertPool) (l Config, err error) {
-	l = Config{}
+	l = Config{
+		rootCAs: rootCAs,
+	}
```

Comment:
This code should be set right after we do `l.Enabled = true` not until then rootCAs should be referenced inside `l = Config{}`